src/components/notebook/habitat-panel.scss

```diff
@@ -2,11 +2,96 @@
 
 .habitat-panel {
   display: flex;
-  justify-content: center;
+  flex-direction: column;
   align-items: center;
   width: 438px;
   height: 304px;
   border-radius: 0 0 10px 10px;
   border: solid 1px $gray-dark;
   border-top: none;
+  font-size: 14px;
+  font-weight: bold;
+
+  .categories{
+    display: flex;
+    flex-direction: row;
+    margin-top: 15px;
+
+    .category {
+      height: 219px;
+      border-right: solid 1px $gray-dark25;
+      padding-right: 9px;
+      margin-right: 9px;
+      margin-bottom: 15px;
+      width: 124px;
+      &:first-child {
+        margin-left: 14px;
+      }
+      &:last-child {
+        border-right: none;
+      }
+
+      .header {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        background-color: $habitat-orange-dark1;
+        width: 124px;
+        height: 24px;
+      }
+
+      .feature-row {
+        display: flex;
+        flex-direction: row;
+        margin: 11px 5px 5px 0;
+        font-size: 14px;
+        font-weight: normal;
+
+        .checkbox {
+          display: flex;
+          justify-content: center;
+          align-items: center;
+          height: 18px;
+          width: 18px;
+          border: solid 2px black;
+          background-color: white;
+          box-sizing: border-box;
+          margin-right: 8px;
+          padding: 0;
+          flex-shrink: 0;
+          cursor: pointer;
+
+          &:hover {
+            background-color: $habitat-green-check-hover;
+          }
+          &:active, &.selected {
+            background-color: $habitat-green-check;
+            border: none;
```

Comment:
I think the <button> tag is putting some kind of default blue border around the checkbox on active, which stays on when selected, and then deselected.
![image](https://user-images.githubusercontent.com/7716653/107317421-7b635480-6a4f-11eb-8f38-e3439732e032.png)

Reply:
Yup, that's the default accessibility selection.  We need to make sure that this is entirely usable with keyboard only.  So all buttons should show their default selection in case a user is trying to navigate with tabs/arrows.

Reply:
You'll notice all buttons in the app show a selection highlight when clicked.